modules/storage-persistent-storage-pv.adoc

```diff
@@ -250,13 +250,13 @@ apiVersion: v1
 kind: PersistentVolume
 metadata:
   name: pv0001
-  annotations:
-    volume.beta.kubernetes.io/mount-options: rw,nfsvers=4,noexec <1>
 spec:
   capacity:
     storage: 1Gi
   accessModes:
   - ReadWriteOnce
+  mountOptions: <1>
+    - nfsvers=4.1
```

Comment:
Here I think it's better to have the same style about the indent with "-"
```
  accessModes:
  - ReadWriteOnce
  mountOptions: <1>
  - nfsvers=4.1
```
Or
```
  accessModes:
    - ReadWriteOnce
  mountOptions: <1>
    - nfsvers=4.1
```
I prefer to use the former.

Reply:
The mountOptions looks good now and looks like the accesModes had bad indentation as well as can be seen in the examples below:
https://docs.openshift.com/container-platform/4.7/storage/understanding-persistent-storage.html#persistent-volumes_understanding-persistent-storage
https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes

So the "-ReadWriteOnce"  line requires two spaces at the beggining as well.

Other than that, looks good to me.

Reply:
@parogui @duanwei33 sure. I will add two spaces before the "-ReadWriteOnce". Thanks

Reply:
@parogui  @duanwei33  Done! PTAL.